my-simple-plugin/include/callbacks.php

```diff
@@ -0,0 +1,38 @@
+<?php 
+/**
+  * This callback function runs third when init action is executed, it's priority collided with another..
+  * callback function but this comes first in execution flow
+  */
+function callback_late() {
+    echo "this function runs third</br>";
+}
+
+/**
+  * This callback function runs second when init action is executed, it has default priority value of 10
+  */
+function callback_normal() {
+    echo "this function runs second</br>";
+}
+
+/**
+  * This callback function runs first when init action is executed, it has priority value of 9
+  */
+function callback_early() {
+    echo "this function runs first</br>";
+}
+
+/**
+  * This callback function runs at last when init action is executed, it has priority is collided with another..
+  * callback function but comes last in execution flow.
+  */
```

Comment:
`// @return void`

Reply:
If there are parameters to any function then even that can be mentioned as @param

Reply:
Same applies for all doc blocks

Reply:
Sure, I'll follow it..